utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -11,6 +11,7 @@
 # ----------------------------------------------------------------------------
 
 import os
+import errno
```

Comment:
Not seeing where this import is getting used?

Reply:
Again, not part of 8f4a746, and I'm pretty sure I've already removed it in an upstream PR. (I haven't rebased this one yet…).